### ISSUE TITLE
Standardize upstream version notation

### ISSUE THREAD

**Issue by OnGle** [turnkeylinux/tracker#679]
While attempting to automate version checking (between turnkey and upstream) I noticed that there is no standard followed for referencing the upstream version. Often the changelogs specify it but not in a reliable manner. Having upstream version specified explicitly in a consistent way would make many automated checks trivial to implement. In most cases versions can be determined by using regex however this is not reliable and in the case of any serious automation could introduce ambiguity, meaning the potential value of the automation in question is limited.

**Comment by JedMeister** [comment]
Good point. FWIW I have tagged this as "documentation" even though it is more than that.

I guess we need to think about what the best way of doing this might be. It could be good to explicitly define how we include upstream version notation in the changelog? Or perhaps we could note it somewhere/somehow else? E.g. perhaps a new file in each repo called "VERSIONS" or "UPSTREAM" or something similar...?

**Comment by JedMeister** [comment]
Whilst I think this is important and should be addressed, I'm moving it to v15.1 milestone...
